modules/covid_vaccine/app/jobs/covid_vaccine/expanded_registration_email_job.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+module CovidVaccine
+  class ExpandedRegistrationEmailJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options expires_in: 1.day, retry: 2
+
+    STATSD_ERROR_NAME = 'worker.covid_vaccine_expanded_registration_email.error'
+    STATSD_SUCCESS_NAME = 'worker.covid_vaccine_expanded_registration_email.success'
+
+    def perform(record_id, email, datestring)
+      submission = CovidVaccine::V0::ExpandedRegistrationSubmission.find_by(id: record_id)
```

Comment:
If you use `find` instead of `find_by` it will raise an ActiveRecord::NotFound which would be just as good and automatically logged by handle_errors. My latest changes included this. It simplifies the guard clauses down to just `submission.email_confirmation_id.present?`

Reply:
Sounds good. Changes the behavior slightly, such that it will retry vs return on a not-found ID (which should never happen regardless, but if it does fine to retry and capture in sentry)

Reply:
yes and might actually be preferred if it's a result of race condition, but you could also handle ActiveRecord::NotFound and not re-raise if it becomes an issue later.